.github/workflows/run_tests.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         export CLARIFAI_USER_ID="$(python scripts/key_for_tests.py --get-userid)"
         export CLARIFAI_PAT="$(python scripts/key_for_tests.py --create-pat)"
         if [ "${{ github.event.pull_request.head.repo.full_name }}" = "${{ github.repository }}" ]; then
-          pytest tests/
+          pytest --cov=.
```

Comment:
Just confirming, this command will run both the tests and coverage right?

Reply:
Yes